docs/en/latest/getting-started.md

```diff
@@ -21,7 +21,7 @@ title: Getting Started
 #
 -->
 
-## Getting Started
+## Getting Started {#apisix}
```

Comment:
I can't accept it for the same result: https://github.com/apache/apisix/pull/4474#discussion_r657713755
It is a scar on the title.